apps/console/src/features/core/hooks/use-routes.tsx

```diff
@@ -163,7 +163,7 @@ const useRoutes = (): useRoutesInterface => {
             getAppViewRoutes(commonConfig.useExtendedRoutes),
             featureConfig,
             allowedScopes,
-            window[ "AppUtils" ].getConfig().organizationName ? false : commonConfig.checkForUIResourceScopes,
```

Comment:
Can we make sure that these changes will not affect the legacy mode i.e cloud offering?

Reply:
This was tested with legacy mode as well. Since `checkForUIResourceScopes` is true for cloud offering, it shouldn't be an issue. However, after completing this, will conduct a test round in cloud offering as well.